composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": "^7.1",
-        "phpstan/phpstan": "^0.11",
```

Comment:
Per https://getcomposer.org/doc/articles/versions.md#tilde-version-range-, 

> The ~ operator is best explained by example: ~1.2 is equivalent to >=1.2 <2.0.0, while ~1.2.3 is equivalent to >=1.2.3 <1.3.0

So specifying tilde 0.11 will still let it update to 0.12, 0.13, etc. until it reaches 1.0 which it will NOT update to. However if you specify tilde 0.11.0, then it will not update to 0.12.

Adding a conflict on 0.12 would only work as long as there is not a 0.13, which tilde 0.11 still allows to update to.